components/MachineForm/index.js

```diff
@@ -1,12 +1,28 @@
+import { useEffect } from "react";
 import {
   FormLabel,
   SubmitButton,
   FormInput,
   FormArea,
   StyledForm,
 } from "./MachineForm.Styled";
+import { useState } from "react";
+import Image from "next/image";
+
+export default function MachineForm({
+  onSubmit,
+  machine,
+  imageThere,
+  imageUrl,
```

Comment:
What is the reason for `imageThere` and `imageUrl`? What does `imageThere` mean?

Reply:
had to try it out a little myself.
and i came to the conclusion that it must have been a relic from when i only got the add form to work.
took imageThere out of the code.
i must have used it to dynamically render the image preview after uploading